x/derivatives/spec/01_concepts.md

```diff
@@ -1,15 +1,73 @@
 # Concepts
 
+The model of Perpetual Futures feature of this module totally follows GMX perpetual futures model.   
+(ref: https://gmxio.gitbook.io/gmx/)
+
+Briefly saying, the tradings on the perpetual futures market are supported by a unique multi-asset pool that earns liquidity prodicers fees from market making, swap fees and levrage trading.   
+Because pool asset will take the counter part of the arbitral trade by a user, users can trade with high levarage and no slippage to the price from oracle with low fee.
+
 ## Pool
 
-- `QuoteTicker` is `USD` by default.
-- Tickers `USD` and `USDC` should be discriminated.
-- Be aware of the difference between `denom` and `ticker`. The former is `uguu` and the latter is `GUU`
+### Liquidity Provider Token
+
+Our liquidity provider token's ticker is `DLP`.   
+In the backend, it has `udlp` denom, which is the micro unit of `DLP`.   
+
+DLP consists of an index of assets used for leverage trading. It can be minted using the assets which the protocol supports and burnt to redeem any index asset. The price for minting and redemption is calculated based on the formulas in the WhitePaper in the section "3.1.1".   
+WhitePaper: https://ununifi.io/assets/download/UnUniFi-Whitepaper.pdf
+
+Fees earned on the platform are directly added to the pool.　Therefore, DLP holders can benefit from them as a reward through the increasement of the DLP price.   
 
-## Liquidity Provider Token
+There's no dynamic change of the minting and redemption fee rate at this moment unlike GMX. It's the static rate which is defined in the protocol. But, it can be modified through the governace voting.
```

Comment:
This is unintended.

https://github.com/UnUniFi/chain/blob/newDevelop/x/derivatives/keeper/lpt.go#L81

Reply:
I modified it.
Please check the above line.